.github/workflows/scalatest-spark3.yml

```diff
@@ -0,0 +1,77 @@
+name: Scala Test with spark3
+
+on:
+  push:
```

Comment:
no nned to keep push and pr
this is nb only run when scheduled
pr can be removed after debug